google-cloud-scheduler/lib/google/cloud/scheduler/v1beta1/cloud_scheduler_client.rb

```diff
@@ -0,0 +1,621 @@
+          #   `projects/PROJECT_ID/locations/LOCATION_ID/jobs/JOB_ID`.
+          # @param options [Google::Gax::CallOptions]
+          #   Overrides the default settings for this call, e.g, timeout,
+          #   retries, etc.
+          # @yield [result, operation] Access the result along with the RPC operation
+          # @yieldparam result [Google::Cloud::Scheduler::V1beta1::Job]
+          # @yieldparam operation [GRPC::ActiveCall::Operation]
+          # @return [Google::Cloud::Scheduler::V1beta1::Job]
+          # @raise [Google::Gax::GaxError] if the RPC is aborted.
+          # @example
+          #   require "google/cloud/scheduler"
+          #
+          #   cloud_scheduler_client = Google::Cloud::Scheduler.new(version: :v1beta1)
+          #   formatted_name = Google::Cloud::Scheduler::V1beta1::CloudSchedulerClient.job_path("[PROJECT]", "[LOCATION]", "[JOB]")
+          #   response = cloud_scheduler_client.resume_job(formatted_name)
+
+          def resume_job \
+              name,
+              options: nil,
+              &block
+            req = {
+              name: name
+            }.delete_if { |_, v| v.nil? }
+            req = Google::Gax::to_proto(req, Google::Cloud::Scheduler::V1beta1::ResumeJobRequest)
+            @resume_job.call(req, options, &block)
+          end
+
+          # Forces a job to run now.
+          #
+          # When this method is called, Cloud Scheduler will dispatch the job, even
+          # if the job is already running.
+          #
+          # @param name [String]
+          #   Required.
+          #
+          #   The job name. For example:
+          #   `projects/PROJECT_ID/locations/LOCATION_ID/jobs/JOB_ID`.
+          # @param options [Google::Gax::CallOptions]
+          #   Overrides the default settings for this call, e.g, timeout,
+          #   retries, etc.
+          # @yield [result, operation] Access the result along with the RPC operation
+          # @yieldparam result [Google::Cloud::Scheduler::V1beta1::Job]
+          # @yieldparam operation [GRPC::ActiveCall::Operation]
+          # @return [Google::Cloud::Scheduler::V1beta1::Job]
+          # @raise [Google::Gax::GaxError] if the RPC is aborted.
+          # @example
+          #   require "google/cloud/scheduler"
+          #
+          #   cloud_scheduler_client = Google::Cloud::Scheduler.new(version: :v1beta1)
+          #   formatted_name = Google::Cloud::Scheduler::V1beta1::CloudSchedulerClient.job_path("[PROJECT]", "[LOCATION]", "[JOB]")
+          #   response = cloud_scheduler_client.run_job(formatted_name)
+
+          def run_job \
+              name,
+              options: nil,
+              &block
+            req = {
+              name: name
+            }.delete_if { |_, v| v.nil? }
+            req = Google::Gax::to_proto(req, Google::Cloud::Scheduler::V1beta1::RunJobRequest)
+            @run_job.call(req, options, &block)
+          end
+
+          def job_path *args
```

Comment:
I would rather see these methods added via a helpers file and include documentation.